pal/system-info/minigame/system-info.ts

```diff
@@ -125,13 +127,13 @@ class SystemInfo extends EventTarget {
 
         this.isXR = false;
 
-        const isPCWechat = WECHAT && this.os === OS.WINDOWS && !minigame.isDevTool;
+        const isPCWechat = (WECHAT || WECHAT_MINI_PROGRAM) && this.os === OS.WINDOWS && !minigame.isDevTool;
         this._featureMap = {
             [Feature.WEBP]: false,      // Initialize in Promise,
             [Feature.IMAGE_BITMAP]: false,
             [Feature.WEB_VIEW]: false,
-            [Feature.VIDEO_PLAYER]: WECHAT || OPPO,
-            [Feature.SAFE_AREA]: WECHAT || BYTEDANCE,
+            [Feature.VIDEO_PLAYER]: WECHAT || WECHAT_MINI_PROGRAM || OPPO,
+            [Feature.SAFE_AREA]: WECHAT || WECHAT_MINI_PROGRAM || BYTEDANCE,
```

Comment:
does wechat mini program support video and safe area ?

Reply:
Yes. But need to adapte in future.